src/Microsoft.Health.Dicom.Core/Features/Store/StoreOrchestrator.cs

```diff
@@ -255,6 +255,17 @@ private static bool TryGetBufferPosition(IByteBuffer buffer, out long position,
             size = fileByteBuffer.Size;
             result = true;
         }
+        else if (buffer is CompositeByteBuffer compositeByteBuffer)
```

Comment:
Is this valid? I forgotten the logic maybe, but it could be that the single frame is in multiple buffers, this won't work right? I had a comment to support that correctly later.

Reply:
In our current case, this won't hit at all. Only if we create sample file using stream or memory byte buffer this hits.

Reply:
I thought of calculating the total size, but we don't use the size. We only use the position and add it with the offset.